package.json

```diff
@@ -84,7 +84,6 @@
     "doctrine": "1.5.0",
     "eslint-import-resolver-node": "^0.3.1",
     "eslint-module-utils": "^2.2.0",
-    "has": "^1.0.1",
```

Comment:
this was unused, but i just fixed a use of hasOwnProperty in master, so this line should be reverted

Reply:
done, thanks for the tip